xbmc/dialogs/GUIDialogBusy.cpp

```diff
@@ -35,6 +35,10 @@ class CBusyWaiter : public CThread
   explicit CBusyWaiter(IRunnable *runnable) :
   CThread(runnable, "waiting"), m_done(new CEvent()),  m_runnable(runnable) { }
   
+  ~CBusyWaiter() {
```

Comment:
Could you please move the curly brace to the next line? This is a kodi coding guideline.

Reply:
Done. I keep forgetting that with Kodi commits.